config/samples/associations-rbac/apm_es_kibana_rbac.yaml

```diff
@@ -0,0 +1,120 @@
+# This file contains an example of Roles, RoleBindings and ServiceAccount which allow the associations to be established
+# between resources living in different namespaces if the access control between resources across namespaces is enabled.
+# This example is only valid if ECK is started with the related option.
+# See https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-operator-config.html.
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kibana-ns
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: elasticsearch-ns
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: apmserver-ns
+---
+# Create a Role at the cluster level to access some Elasticsearch clusters.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: elasticsearch-association
```

Comment:
Do you think we should pre-create this role as part of the operator manifest?

Reply:
My gut feeling is that we should create roles only if there are needed but add it as an example in our documentation if the user wants to enable access control. That being said I'm happy to pre-create it if you think it's worth it.

Reply:
👍  let's not include it for now and maybe do it later depending on users feedback.